Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
     ]
 
     args << "-nomake" << "examples" if build.without? "examples"
 
-    args << "-plugin-sql-mysql" if build.with? "mysql"
-    args << "-plugin-sql-psql" if build.with? "postgresql"
+    if build.without? "plugins"
+      use_plugin = "qt"
+      args << "-qt-sql-sqlite"
+      inreplace "qtbase/configure" do |s|
+        s.gsub! "CFG_JPEG=plugin", "CFG_JPEG=yes"
+        s.gsub! "CFG_GIF=plugin", "CFG_GIF=yes"
```

Comment:
I suggest something like this.
Diff to steve3d's 3rd commit. https://github.com/yezezey/homebrew-core/commit/28f46e6c4db62946dbebb2adceaa8964042890c9

I am not a Ruby person and it's not well tested because building Qt is time consuming...